portfoliostyle.css

```diff
@@ -0,0 +1,438 @@
+
```

Comment:
Remove excess whitespace to condense code and keep it consistent :)

Reply:
😅